microsoft-edge/webview2/how-to/context-menus.md

```diff
@@ -0,0 +1,920 @@
+                                          labelString.c_str()));
+                }
+            }
+            else if (
+                kind == COREWEBVIEW2_CONTEXT_MENU_ITEM_KIND_CHECK_BOX ||
+                kind == COREWEBVIEW2_CONTEXT_MENU_ITEM_KIND_RADIO)
+            {
+                if (isEnabled)
+                {
+                    if (isChecked)
+                    {
+                        CHECK_BOOL(AppendMenu(hPopupMenu, 
+                                              MF_CHECKED | MF_STRING, 
+                                              commandId, 
+                                              labelString.c_str()));
+                    }
+                    else
+                    {
+                        CHECK_BOOL(AppendMenu(hPopupMenu, 
+                                              MF_BYPOSITION | MF_STRING,
+                                              commandId, 
+                                              labelString.c_str()));
+                    }
+                }
+                else
+                {
+                    if (isChecked)
+                    {
+                        CHECK_BOOL(AppendMenu(hPopupMenu, 
+                                              MF_CHECKED | MF_GRAYED | MF_STRING, 
+                                              commandId, 
+                                              labelString.c_str()));
+                    }
+                    else
+                    {
+                        CHECK_BOOL(AppendMenu(hPopupMenu, 
+                                              MF_GRAYED | MF_STRING, 
+                                              commandId, 
+                                              labelString.c_str()));
+                    }
+                }
+            }
+            else if (kind == COREWEBVIEW2_CONTEXT_MENU_ITEM_KIND_COMMAND)
+            {
+                if (isEnabled)
+                {
+                    CHECK_BOOL(AppendMenu(hPopupMenu,
+                                          MF_BYPOSITION | MF_STRING, 
+                                          commandId, 
+                                          labelString.c_str()));
+                }
+                else
+                {
+                    CHECK_BOOL(AppendMenu(hPopupMenu, 
+                                          MF_GRAYED | MF_STRING, 
+                                          commandId, 
+                                          labelString.c_str()));
+                }
+            }
+        }
+    }
+```
+
+---
+
+
+<!-- ====================================================================== -->
+## Adding menu items to a context menu
+
+You can:
+*  Add default menu items to a custom context menu.
+*  Add custom menu items to a default context menu, as shown in an example code listing below.
+
+<!-- -------------------------------------------------- -->
+### Adding default menu items to a custom context menu
```

Comment:
This part is already mentioned above in the "Adding custom context menu" section

Reply:
@yildirimcagri - merged section "Adding default menu items to a custom context menu" up into section "Adding a custom context menu".